tests/examples/test_multiprocess_redis_database/test_split.py

```diff
@@ -10,8 +10,8 @@
 A correct implementation would use some mechanism to avoid this inter-parallel-test
 key conflict problem.
 """
-import random
-import time
+# import random
+# import time
```

Comment:
commented code

Reply:
see below in this file, the imports are there to make the further down commented code that i'd like to uncomment once https://github.com/schireson/pytest-mock-resources/pull/129 is merged

Reply:
can you delete these imports that would be obvious once the code below is uncommented?